apis/definitions/schemas/address.yaml

```diff
@@ -22,16 +24,12 @@ definitions:
       updatedBy:
         type: string
       updatedDate:
-         type: string
-         format: yyyy-MM-dd HH:mm:ss:SSSZZZZ
+        type: string
+        format: yyyy-MM-dd HH:mm:ss:SSSZZZZ
       createdDate:
-  	 type: string
-	 format: yyyy-MM-dd HH:mm:ss:SSSZZZZ
-      isDeleted:
-          type: boolean
-      createdBy:
-          type: string
-      userId:
-          type: string	  
-     
-     
+  	    type: string
```

Comment:
please validate all files before raising PR, this indentation is not valid.

Reply:
@SMYALTAMASH now it is proper. please check.